content/news/052/index.md

```diff
@@ -80,8 +80,8 @@ We will switch this to an actual @gamedev.rs address in the future. Please tell
 
 ### [Untitled Pixel Wizards Game][pixel-wizards]
 
-{{ embed_video(type="video/mp4", src="untitled-pixel-wizards-game.mp4",
-caption="Enemies now perceive, pursue and attack.. and occasionally get burned to death.") }} 
+{{ embed_video(type="video/mp4", src="untitled-pixel-wizards-game.mp4") }}
+*Enemies now perceive, pursue, and attack... and occasionally get burned to death.*
```

Comment:
Ya, let me grab some screenshots

Reply:
<img width="772" alt="image" src="https://github.com/rust-gamedev/rust-gamedev.github.io/assets/14791619/488309df-4db7-4c90-b237-043cb0a0cd9f">

<img width="772" alt="image" src="https://github.com/rust-gamedev/rust-gamedev.github.io/assets/14791619/826dec76-c57c-428b-826f-f85fd2564bf7">

Reply:
So I think I just did it for the spacing, since it becomes more consistent with the other captions. I think the play here would be use the hacky captions for now until the CSS is improved, and we'll then need to retroactively go through each article and change it out :P